src/graph/planner/plan/PlanNode.cpp

```diff
@@ -370,6 +370,13 @@ void PlanNode::releaseSymbols() {
   }
 }
 
+void PlanNode::updateSymbols() {
+  auto symTbl = qctx_->symTable();
+  for (auto out : outputVars_) {
+    out && symTbl->updateWrittenBy(out->name, out->name, this);
```

Comment:
if (out != nullptr)

Reply:
if out is nullptr, `out && updateWrittenBy` updateWrittenBy will be cut off

Reply:
It's not a clear code style.

Reply:
thanks, fixed